vme/src/char_data.cpp

```diff
@@ -351,17 +351,17 @@ void char_data::incrementLevel()
     m_points.incrementLevel();
 }
 
-ubit8 char_data::getPosition() const
+Position_e char_data::getPosition() const
 {
     return m_points.getPosition();
 }
 
-void char_data::setPosition(ubit8 value)
+void char_data::setPosition(Position_e value)
 {
     m_points.setPosition(value);
 }
 
-ubit8 *char_data::getPositionPtr()
+Position_e *char_data::getPositionPtr()
```

Comment:
So what happens in the DIL functions that manipulate the position? Are you certain that Position_e is a one byte value?

Reply:
No the size is usually left up to the compiler and can change between enums. Where would be a good place to look where that position ptr is used, its hard to find things when they disappear into void *'s :-)